src/Lykke.Job.PayTransactionHandler.Services/Wallets/WalletsScanService.cs

```diff
@@ -102,17 +90,17 @@ public class WalletsScanService : IScanService
 
                                 createRequest = tx.ToCreateRequest(txDetails, _bitcoinNetwork);
 
-                                await _log.WriteInfoAsync(nameof(ExecuteAsync), new
+                                _log.Info("New transaction detected", new
                                 {
                                     Hash = tx.Id,
                                     txDetails.FirstSeen
-                                }.ToJson(), "New transaction detected");
+                                });
 
                                 await _payInternalClient.CreatePaymentTransactionAsync(createRequest);
                             }
                             catch (Exception ex)
                             {
-                                await _log.WriteErrorAsync(nameof(ExecuteAsync), createRequest?.ToJson(), ex);
+                                _log.Error(ex, context: createRequest);
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.